packages/tradershub/src/features/cfd/components/MT5PlatformsList/MT5PlatformsList.tsx

```diff
@@ -15,14 +15,15 @@ type TMT5PlatformsListProps = {
 const MT5PlatformsList = ({ onMT5PlatformListLoaded }: TMT5PlatformsListProps) => {
     const { isFetching } = useAuthorize();
     const { uiState } = useUIContext();
+    const { accountType } = uiState;
     const activeRegulation = uiState.regulation;
     const { areAllAccountsCreated, data, isFetchedAfterMount } = useSortedMT5Accounts(activeRegulation ?? '');
     const { data: activeTradingAccount } = useActiveTradingAccount();
     const { isEU } = useRegulationFlags();
     const invalidate = useInvalidateQuery();
 
     const hasMT5Account = useMemo(() => {
-        return data?.some(account => account.is_added);
+        return data?.some(MT5Account => MT5Account.is_added);
```

Comment:
const Mt5AccountList = data.find(MT5Account =>MT5Account.is_virtual === activeTradingAccount?.is_virtual)

Reply:
@mahdiyeh-deriv I'm not sure what you are referring to, can you elaborate more about it? thanks 🙏

Reply:
but if we wanna do data.find it will return only one element which here we need an array, please correct me if I'm wrong  🙏